linguatec_lexicon/importers.py

```diff
@@ -0,0 +1,457 @@
+import pandas as pd
+import sys
+
+from linguatec_lexicon.models import (Entry, GramaticalCategory, Word,
+                                      Lexicon, DiatopicVariation, Example,
+                                      VerbalConjugation, ImportsInfo)
+
```

Comment:
Extra blank line. You can use "Sort imports" from VS to organize imports (right-click on the code)

Reply:
Organized and removed extra line, thank you for the tip